.eslintrc

```diff
@@ -13,6 +13,11 @@
     "strict": 0,
     "linebreak-style": 0,
     "implicit-arrow-linebreak": 0,
-    "object-curly-newline": 0
+    "object-curly-newline": 0,
```

Comment:
I don't think crippling the LINTer to make the issues go away is a good practice.

Reply:
Fixed all eslint issue